### PR TITLE
Round to nearest card

### PR DESCRIPTION
**Pull request by themoch**
## Description
Sometimes it is helpful to provide the nearest card that the average equates to.  This option allows the Host to show what the average point value equates to in the current deck.

### Code Details
* Created toggle in host.js for option value
* Created function `getNearestCard` which returns the card closest to the `voteData.average`
* Created DOM for option
* Created DOM for results to host
* Created DOM for results to joined user
* LESS styling

### Screen Shot
screen shot 2017-04-06 at 4 08 21 pm

### CODE REVIEW

**Comment by ravjot** [comment]
@jackwanders / @twisterghost  ☝️

File: public/javascripts/host.js [hunk 1]
```diff
@@ -133,11 +159,17 @@ var processVotes = function() {
...
 	voteData.spread = maxCardIdx - minCardIdx;
-	voteData.average = voteData.numVotes === 0 ? 0 : voteData.total/voteData.numVotes;
-	if(voteData.average > 0.5) {
-		voteData.trueAverage = voteData.average;
-		voteData.average = Math.round(voteData.average);
+
+	voteData.average = voteData.numVotes === 0 ? 0 : voteData.total / voteData.numVotes;
+
+	if (nearestRounding) {
+    voteData.nearestCard = getNearestCard(voteData.average, deck);
```
Comment (twisterghost) [inline public/javascripts/host.js]: Can you clean up the indentation here and on line 201?

**Comment by twisterghost** [comment]
Looks good. Minor style comment but ready to go otherwise.

**Comment by themoch** [comment]
@twisterghost done and done

**Comment by twisterghost** [comment]
Awesome. I'll pull it down to test and whatnot and merge if it all looks good.

**Pull request merged**